xrpl4j-core/src/test/java/org/xrpl/xrpl4j/codec/binary/types/BaseSerializerTypeTest.java

```diff
@@ -58,7 +58,25 @@ void fixtureTests(ValueTest fixture) throws IOException {
       SerializedType<?> serialized = serializedType.fromJson(value);
       if (fixture.type().equals("Amount")) {
         AmountType amountType = (AmountType) serialized;
-        assertThat(amountType.isPositive()).isEqualTo(!fixture.isNegative());
+        // Special-case for zero-value IOU amounts.
+        if (
+          !amountType.toJson().has("mpt_issuance_id") &&// <-- Not MPT
```

Comment:
Need this one space to fix Checkstyle. Sorry about the typo.

```suggestion
          !amountType.toJson().has("mpt_issuance_id") && // <-- Not MPT
```